scripts/jenkinsProductionFiles.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# jenkinsProductionFiles.sh by Andrzej Pogonowski
+
+# Steps
+# 1. Removes all of the build and dependency directories (build, pack, node_modules)
+# 2. Copies all of the source files
+# 3. Copes configuration files and other necessary files
+
+# Cleanup
+rm -r pack
+rm -r build
+rm -r node_modules
+mkdir pack
+chmod -R 755 pack
+
+# Copy needed directories
+cp -rf ./src ./pack/src
+cp -rf ./cypress ./pack/cypress
+cp -rf ./.storybook ./pack/.storybook
+cp -rf ./envConfig ./pack/envConfig
+cp -rf ./public ./pack/public
+
+# Copy the needed files in the root directory
+cp package.json ./pack
```

Comment:
Our CI/CD should only ever run `npm ci` meaning it'll use ` package-lock.json` and not `package.json` so this shouldn't be explicitly needed for production.

Reply:
Is the `package.json` not needed in the RPM though? The scripts would be needed for starting the application.